updateinfo_valid.xml

```diff
@@ -0,0 +1,172 @@
+<updates>
```

Comment:
Looks like you have added this file by mistake.

Reply:
Sorry, I delete the file.